ServerSide/yoty/Migrations/YotyContextModelSnapshot.cs

```diff
@@ -0,0 +1,268 @@
+﻿// <auto-generated />
+using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+using Yoty.Data;
+
+namespace yoty.Migrations
```

Comment:
should be YOTY.Service.Data.Migration

Reply:
@tzachihakmon right, I’ll fix all namespaces